libs/community/tests/unit_tests/vectorstores/test_sqlserver.py

```diff
@@ -32,6 +48,21 @@
 and schema_name(schema_id) = '%s'"""
 
 
+# Combine all test cases into one list with additional debugging
+FILTERING_TEST_CASES: List[Any] = []
+for filterList in [
+    TYPE_1_FILTERING_TEST_CASES,
```

Comment:
nit: names of these test cases should also be fixed. I dont know what is TYPE_1_FILTERING_TEST_CASES

Reply:
This is not added by us Aniruddh. Thes are the test cases already present in Langchain package and being used by other vector stores too.